object_bbox_topic: /euclidean_clustering_node/bbox
margin: 0.5
resolution: 0.20
map_height: 200
map_width: 200
height_offset: -2.5
buffer_length: 5
map_frame: map
robot_frame: base_link